com.ticktick.TickTick.yml

```diff
@@ -0,0 +1,77 @@
+app-id: com.ticktick.TickTick
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: ticktick
+rename-desktop-file: ticktick.desktop
+rename-icon: ticktick
```

Comment:
you install the icons with the right name.
```suggestion
```

But then you need to change the icon name in the `.desktop`